src/widgetastic_patternfly5/charts/boxplot_chart.py

```diff
@@ -0,0 +1,42 @@
+from widgetastic.widget import ParametrizedLocator
+from widgetastic.widget import View
+
+from .legend import Legend
+from widgetastic_patternfly5.charts.line_chart import LineChart
+
+
+class BoxPlotChart(LineChart):
+    """Represents the Patternfly Line Chart.
+
+    https://patternfly-react-main.surge.sh/charts/box-plot-chart#embedded-legend
+
+    Args:
+        id: If you want to look the input up by id, use this parameter, pass the id.
+        locator: If you have specific locator else it will take pf-chart.
+    """
+
+    ROOT = ParametrizedLocator("{@locator}")
+
+    X_AXIS_LABELS = "(.//*[name()='g' and *[name()='line'] and *[name()='g']])[1]//*[name()='text']"
+    Y_AXIS_LABELS = "(.//*[name()='g' and *[name()='line'] and *[name()='g']])[2]//*[name()='text']"
+
+    TOOLTIP = (
+        ".//*[name()='g' and .//*[name()='g' and "
+        ".//*[name()='text' and contains(@id, 'legend-labels')]]]"
+    )
+
+    TOOLTIP_X_AXIS_LABLE = ".//*[name()='text']"
+    TOOLTIP_LABLES = ".//*[name()='g']/*[name()='text' and not(contains(@id, 'legend-label'))]"
+    TOOLTIP_VALUES = ".//*[name()='g']/*[name()='text' and contains(@id, 'legend-label')]"
+
+    _legends = View.nested(Legend)
+
+    def __init__(self, parent=None, id=None, locator=None, logger=None):
```

Comment:
Still, some stuffs are common which you don't need to specify here.
whatever common we don't need to specify in inheritance ^^